ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/LearningCurveMultiDataset.java

```diff
@@ -240,21 +239,20 @@ public static void getLearningCurve(Vector<Data> trainDataSet, Vector<Data> test
         }
     }
 
-    /*
+    /**
```

Comment:
The only change I made here was to make the comment a legal javadoc comment. I have no idea what the text actually means. I don't even see a place where the fold sizes are specified in this method. I also turned off the useless output to standard out so we can wade through less text trying to find results. I don't think we are using n-fold methodology with NER at all, we are doing the auto-convergence or specifying the number of iterations.